be/src/common/config.h

```diff
@@ -252,7 +252,7 @@ CONF_Int64(index_stream_cache_capacity, "10737418240");
 // Cache for stoage page size
 CONF_String(storage_page_cache_limit, "0");
 // whether to disable page cache feature in storage
-CONF_Bool(disable_storage_page_cache, "true");
```

Comment:
I think the configuration can be removed. And use storage_page_cache_limit instead.